### ISSUE TITLE
plist_options audit failures

### ISSUE THREAD

**Issue by ilovezfs** [Homebrew/homebrew-core#4029]
```
Please set plist_options when using a formula-defined plist.
```

These are formulae in need of `plist_options`:
- [x] automysqlbackup
- [x] beansdb
- [x] boot2docker
- [x] fuseki
- [x] kettle
- [x] logrotate
- [x] mailhog
- [x] mongo-orchestration
- [x] mysql-cluster
- [x] nrpe
- [x] oauth2_proxy
- [x] polipo
- [x] postgres-xc
- [x] proftpd
- [x] riemann
- [x] uptimed

**Comment by ilovezfs** [comment]
@DomT4 I think we should consider making this strict-failure only if no one wants to help.

**Comment by DomT4** [comment]
That's most of Homebrew though. People ignore tracking issues like the issues are a jar full of poison cookies. It's on my list, but the list ain't particularly short.

https://github.com/Homebrew/homebrew-core/pull/4207 fixes `riemann` at least.

**Comment by ilovezfs** [comment]
@DomT4 I'm just generally annoyed by making things non-strict failure when the preexisting ones haven't been fixed first.

**Comment by ilovezfs** [comment]
Only two left! mysql-cluster and nrpe.

**Comment by harshnarang8** [comment]
@ilovezfs mysql-cluster and nrpe don't give any output for brew audit --strict. Maybe they were fixed in a commit?

**Comment by lwe** [comment]
@ilovezfs, yes can confirm what @harshnarang8 said too:

* mysql-cluster was fixed in 94b9cc70
* nrpe in 84d1763d

**Comment by ilovezfs** [comment]
Hurray! Thanks to everyone who helped. Onward to https://github.com/Homebrew/homebrew-core/issues/10839.
